### ISSUE TITLE
Create a backup mirror for the winesapOS repository

### ISSUE THREAD

**Issue by AireagoirSteam** [LukeShortCloud/winesapOS#577]
Tried updating packages using buah and I think the repo is down since nothing can connect to it.

Would be useful if could do an archive of the repo using the wayback machine since if it ever goes down that could be used as a fallback.

**Comment by LukeShortCloud** [comment]
@AireagoirSteam Thanks for reporting this issue! What version of winesapOS are you using? Valve recently changed all of their repository names without telling the community which lead to lots of issues with Linux distributions based on SteamOS 3. I have fixed this in winesapOS 3.2.1. Use the "winesapOS Upgrade" desktop shortcut to upgrade if you are on an older version.

I cannot reproduce the issue on my end. You can try downloading a small package from our repository to ensure you can connect to the winesapOS server:

```
wget https://winesapos.lukeshort.cloud/repo/winesapos/x86_64/zerotier-gui-git-1.3.0.r84.44626a5-1-any.pkg.tar.zst
```

Either way, I really appreciate the idea of using the Wayback Machine to archive our repositories! I'll look into that.

**Comment by LukeShortCloud** [comment]
Related: https://github.com/LukeShortCloud/winesapOS/issues/474

**Comment by LukeShortCloud** [comment]
I have enabled directory listing of the winesapOS repository. It is now possible to make a backup of the repository.

I am in the process of backing it up to the Wayback Machine and will add additional documentation about how to do this before closing this GitHub Issue. I found some starting good hints here:

https://webapps.stackexchange.com/questions/115369/how-to-archive-the-whole-website

Here is an example of how to access an old package: https://web.archive.org/web/20230610204532/https://winesapos.lukeshort.cloud/repo/winesapos-3.1.0/x86_64/linux-steamos-headers-5.13.0.valve10.1-2-x86_64.pkg.tar.zst
